aiida_raspa/utils/base_parser.py

```diff
@@ -21,21 +21,19 @@ def float(number):  # pylint: disable=redefined-builtin
 # manage block of the first type
 # --------------------------------------------------------------------------------------------
 BLOCK_1_LIST = [
-    (re.compile("Average Volume:"), "cell_volume", (1, 2, 4), 0),
-    (re.compile("Average Pressure:"), "pressure", (1, 2, 4), 0),
-    (re.compile("Average temperature:"), "temperature", (1, 2, 4), 0),
-    (re.compile("Average Density:"), "adsorbate_density", (1, 2, 4), 0),
-    (re.compile("Total energy:$"), "total_energy", (1, 2, 4), 0),
-    (re.compile("Average Heat Capacity"), "framework_heat_capacity", (1, 2, 4), 0),
-    (re.compile("Heat of desorption:"), "heat_of_desorption", (1, 4, 3), 4),
-    (re.compile("Enthalpy of adsorption:"), "enthalpy_of_adsorption", (1, 4, 3), 4),
-    (re.compile(".*Total enthalpy of adsorption from components and measured mol-fraction"),
-     "enthalpy_of_adsorption_total_molfrac", (1, 4, 3), 0),
+    #("Average temperature:", "temperature", (1, 2, 4), 0), # misleading property!
+    #("Average Pressure:", "pressure", (1, 2, 4), 0), # misleading property!
+    ("Average Volume:", "cell_volume", (1, 2, 4), 0),
+    ("Average Density:", "adsorbate_density", (1, 2, 4), 0),
+    #("Average Heat Capacity", "framework_heat_capacity", (1, 2, 4), 0), # misleading property!
+    ("Enthalpy of adsorption:", "enthalpy_of_adsorption", (1, 4, 3), 4),
+    ("Tail-correction energy:", "tail_correction_energy", (1, 2, 4), 0)
+    #("Total energy:$", "total_energy", (1, 2, 4), 0), # not important property!
```

Comment:
Can you please explain why those properties are misleading/not important? Otherwise, it is fine with me.

Reply:
* Heat capacity is meaningless for rigid particles/frameworks
* Temperature and pressure are giving 0.0: not clear why
* The total energy of a FF system is useless: anyway, I already parsed the host/ads and ads/ads contributes of the energy that can be useful in case of comparing interactions.